### PR TITLE
Connect to API

### PR DESCRIPTION
**Pull request by Kakalanp**
Adding API functions that allow you to:
- See the books stored in the API by displaying them on screen.
- Add books to said API by filling the required fields and pressing a button.
- Remove books from said API, those books won't display on the screen never again!

### CODE REVIEW

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
+import { combineReducers, createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
 
 import booksReducer from './books/books';
 import categoriesReducer from './categories/categories';
 
 const rootReducer = combineReducers({ books: booksReducer, categories: categoriesReducer });
 
-const store = createStore(rootReducer);
+const store = createStore(rootReducer, applyMiddleware(thunk));
```
Comment (mateo951) [inline src/redux/configureStore.js:9]: Well done 🚀 `redux-thunk` is added to the project as a package and used as a middleware 🔥

File: src/redux/books/books.js [hunk 2]
```diff
@@ -14,21 +19,69 @@ function deleteBook(id, state = []) {
...
+
+export const removeBook = ({ id }) => async (dispatch) => {
+  await fetch(`https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/VpD4DoliMdQTvZBLcSES/books/${id}`, {
+    method: 'DELETE',
+  })
+    .then(() => dispatch({ type: REMOVE_BOOK, id }));
+};
+
+export const fetchBooks = () => async (dispatch) => {
+  await fetch('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/VpD4DoliMdQTvZBLcSES/books')
```
Comment (mateo951) [inline src/redux/books/books.js:84]: All Bookstore API calls are done with `fetch` OR `axios` ✔

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @Kakalanp, 

📝👌 **Your project highlights** 🔥😎 
- Nice PR 🔥
- External API was implemented ✔

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by mateo951** [comment]
Please be sure that your Github actions get triggered. It's important that on every project you have the linters setup and running checks when you create the PR 🧐

**Review by mikethreels — APPROVED** [review]
Hi Joaquin, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kakalanp** [comment]
Thanks for the approval. Merging right away! 💯

**Pull request merged**